Makefile

```diff
@@ -0,0 +1,37 @@
+.DEFAULT_GOAL := help
+
+#------------------------------------------------------------------------------
+# Purpose:
+#   Makefile that allows one to docker build, install openmct and start openmct
+#
+# Author of this Makefile: 
+#  Haisam Ido<haisam.ido@gmail.com>
+#
+# Dependencies:
+#   make, docker, docker-compose
+#------------------------------------------------------------------------------
+
+openmct-install: ## Install openmct on a docker image. This uses ./Dockerfile
+	docker build -t openmct:latest .
```

Comment:
Looks like all of those make commands could go into package.json run scripts, so it would require using new tool without good justification.

Reply:
I am not sure I agree with that, but I appreciate your feedback.